<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentsForm_template</class>
 <widget class="QDialog" name="CommentsForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="The title of the dialog to choose the comments">Comments</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="commentsTextEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="okPushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
